.gitignore

```diff
@@ -0,0 +1,6 @@
+/venv
+*.py~
+*.pyc
+*.gitignore~
```

Comment:
There were this files with the extension .py~ created when ever i ran some a file so i added them here to disinclude them